packages/test/local-server-tests/src/test/noDeltaStream.spec.ts

```diff
@@ -111,13 +111,11 @@ describe("No Delta Stream", () => {
 
         assert.strictEqual(container.connected, true, "container.connected");
         assert.strictEqual(container.clientId, "storage-only client", "container.clientId");
-        assert.strictEqual(container.readonly, true, "container.readonly");
-        assert.strictEqual(container.readonlyPermissions, true, "container.readonlyPermissions");
```

Comment:
Since it was in storage-only mode, I was unable to replicate the test for `readonlyPermissions` using the `readOnlyInfo` property

Reply:
should delete both these lines then since it duplicates the next check

Reply:
I think storage-only should imply read-only, so appropriate check should be possible